hathor/consensus/transaction_consensus.py

```diff
@@ -296,7 +296,7 @@ def check_conflicts(self, tx: Transaction) -> None:
         for candidate in candidates:
             tx_meta = candidate.get_metadata()
             if not tx_meta.voided_by:
-                candidate.update_accumulated_weight(stop_value=meta.accumulated_weight)
+                candidate.update_accumulated_weight(stop_value=work_to_weight(meta.accumulated_weight))
```

Comment:
I think we should keep passing work (instead of weight) because it is the actual way we are calculating the accumulated weight and there's no reason to convert to weight with loss of precision.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/1193/commits/f42c4a7e07adfbdd7a8fce8fb73a488a2a5fa8f2

Reply:
Thanks!